vendor/github.com/skeema/tengo/flavor.go

```diff
@@ -16,6 +16,7 @@ const (
 	VendorMySQL
 	VendorPercona
 	VendorMariaDB
+	VendorUbuntu
```

Comment:
Is Ubuntu actually changing any functionality in their distribution? (Vendors are meant to represent MySQL distributions/forks with substantial feature changes.)

btw any changes in vendor/ need to be made to the source repos, in this case https://github.com/skeema/tengo/, otherwise the next invocation of `dep ensure` will overwrite them. I will probably switch from `dep` to Go 1.11+ Modules in the near future, but the timing is still TBD.

Reply:
Ubuntu does not change any functionality in their distribution. I added VendorUbuntu here to minimize the tengo/flavor.go ParseVendor code changes as Ubuntu MySQL package build hardcoded @@version_comment MySQL variable as `(Ubuntu)`

Reply:
Ah, thanks! I can add some conversion code in `ParseVendor()` to treat that as VendorMySQL. I just need to test some things first, to see if they similarly change the version comments for their distribution of MariaDB as well...

Reply:
FWIW, when installing mysql with homebrew on mac os:
```
mysql> select @@version, @@version_comment;
+-----------+-------------------+
| @@version | @@version_comment |
+-----------+-------------------+
| 5.7.26    | Homebrew          |
+-----------+-------------------+
```
and mariadb:
```
mysql> select @@version,@@version_comment;
+-----------------+-------------------+
| @@version       | @@version_comment |
+-----------------+-------------------+
| 10.3.16-MariaDB | Homebrew          |
+-----------------+-------------------+
```

so I need to define the flavor in `.skeema` to get the right one.

Reply:
Good to know, thanks! I'll tweak the logic a bit further to handle this more generically.